salt/utils/verify.py

```diff
@@ -218,12 +219,11 @@ def verify_env(dirs, user, permissive=False, pki_dir='', skip_extra=False, root_
             continue
         if not os.path.isdir(dir_):
             try:
-                cumask = os.umask(18)  # 077
-                os.makedirs(dir_)
+                with salt.utils.files.set_umask(0o022):
```

Comment:
Note that while this seems like an incorrect change, 022 is octal for 18. The _original_ umask here was 077, but this was changed in 7eb909871f8ca3d31379d392c6c1d1f8fc106193 many years ago. The comment was just not modified to reflect the change.